src/mmw/js/src/draw/views.js

```diff
@@ -568,6 +573,11 @@ var ResetDrawView = Marionette.ItemView.extend({
     },
 
     resetDrawingState: function() {
+        App.getLeafletMap().eachLayer((l) => {
+            if (l.id && l.id === 'rwd-origin-point') {
+                App.getLeafletMap().removeLayer(l);
+            }
+        });
```

Comment:
Just pointing out that the machinery to show/hide layers depending on different state may already exist. I'd have to verify that though.